metadata_mapper/mappers/flickr/flickr_mapper.py

```diff
@@ -0,0 +1,85 @@
+from ..mapper import Record, Vernacular
+import json
+
+
+class FlickrRecord(Record):
+    def UCLDC_map(self):
+        return {
+            "calisphere-id": self.legacy_couch_db_id.split('--')[1],
+            "isShownAt": self.map_is_shown_at,
+            "isShownBy": self.map_is_shown_by,
+            "date": self.map_date,
+            "description": self.map_description,
+            "format": ["Photo"],
+            'identifier': [self.source_metadata.get("id"), self.url_image],
```

Comment:
I'm not sure either. Some of the flickr-fetched mappers do have values like this in the Solr data:

```                        'identifier': ['40982751',
                                       '14_001411.tif',
                                       '14_001411'],
```

But that's just a filename, not a URL, so I'm having trouble explaining this one. Additionally, that first value also doesn't match the `id` value, which, in this case, is `c9a8add342ed62e287e53f18c5b633f9`. The oddball single quotes are giving debugging / troubleshooting vibes too, but I don't know how I got there. Apologies!

I will take a deeper look the identifier values on these flickr mappers and try to figure this out.

Reply:
I removed the `self.url_image` bit. This configuration only impacts `flickr.flickr`, which is a little unreliable to validate at the moment. I'm going to close this because this specific issue is resolved, but have switch this back to draft until we can successfully validate each of the mappers.